sandbox-web/src/main/java/com/exadel/sandbox/team5/ReviewRestController.java

```diff
@@ -0,0 +1,35 @@
+package com.exadel.sandbox.team5;
+
+import com.exadel.sandbox.team5.entity.DemoEntity;
+import com.exadel.sandbox.team5.entity.Review;
+import com.exadel.sandbox.team5.service.ReviewService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/discount")
+@RequiredArgsConstructor
+public class ReviewRestController {
+    private final ReviewService service;
+
+//    Add endpoint to find reviews for discount (/api/discount/{discountId}/reviews)
+
+    @GetMapping("/{id}")
+    public List<Review> save(@PathVariable Long id) {
+        if (id != null && service.getById(id) != null) {
+            System.out.println("Hello");
+            return service.getAll().stream().
+                    filter(entity -> entity.getDiscount().getId().equals(id)).
```

Comment:
Bad code formatting, better to use following style (dots with methods at the same line):
```
service.getAll().stream()  
                  .filter(entity -> entity.getDiscount().getId().equals(id))
                  .collect(Collectors.toList());
```